api/libproject/devstart.rb

```diff
@@ -833,6 +833,11 @@ def build_review_all_events(cmd_name, *args)
       ->(opts, v) { opts.bq_dataset = v},
       "BQ dataset. Required."
   )
+  op.add_option(
+    "--domain-token [domain-token]",
+    ->(opts, v) { opts.domain_token = v},
+    "Generate for domain-token. Required."
+  )
```

Comment:
you also need to pass the arg in the script call below on line 847

Reply:
Done, thanks.